### ISSUE TITLE
Error: with `@nuxt/eslint-config` eslint marks composables as "undefined" when used inside another composable

### ISSUE THREAD

**Issue by stefanobartoletti** [nuxt/eslint#314]
When using `@nuxt/eslint-config`, `eslint` uncorrectly marks composables as `undefined` when they are used inside another composable.

According to how Nuxt works, composables are auto-imported (and they work when executed), so eslint should not report this error.

A simple test composable that uses internal functions is 

```js
export const useComposable = () => {
  const myConst = ref('hello');

  onMounted(() => {
    console.log(myConst.value);
  });
};

```

And it is available in this minimal demo https://stackblitz.com/edit/github-nxygas?file=package.json,.eslintrc.cjs

(please note that you have to `CTRL + C` and run `npm run lint` to see the errors)

**Comment by jeannen** [comment]
Same issue for me

**Comment by m-shum** [comment]
Still having this issue. Did anyone find a workaround?
